tasks1.js

```diff
@@ -0,0 +1,195 @@
+// 1
+const width = 10;
+const height = 20;
+const rectangleArea = width * height;
+console.log(rectangleArea);
+
+// 2
+let firstNumber = 100;
+let secondNumber = 200;
+let tempNumber = firstNumber;
+firstNumber = secondNumber;
+secondNumber = tempNumber;
+console.log(firstNumber, secondNumber);
+
+// 3
+const physicsGrade = 4
+const mathGrade = 3
+const englishGrade = 5
+
+const avarageGrade = (physicsGrade + mathGrade + englishGrade)/3;
+
+if (avarageGrade >=4 ) {
+  console.log('avarage grade is ' + avarageGrade + ' congrats!')
+}
+
+// 4
+const currentAge = 30;
+const retirenmentAgeInPoland = 67;
+let yearsLeftToRetirenment;
+
+yearsLeftToRetirenment = retirenmentAgeInPoland - currentAge;
+
+console.log(yearsLeftToRetirenment);
+
+// 5
+const isPayingVAT = true;
+let VATRateInPoland = 0.23;
+const incomeTaxRate = 0.12;
+let monthlyIncome = 3500;
+let taxToPay = 0;
+
+if (isPayingVAT) {
+  taxToPay = monthlyIncome * VATRateInPoland;
+}
+
+taxToPay = taxToPay + monthlyIncome * incomeTaxRate;
+
+console.log('tax to pay: ' + taxToPay);
+
+// 6
+const isSunny = false
+if (isSunny) {
+  console.log("Don't forget your sunglasses!")
+} else {
+  console.log("You might need an umbrella.");
+}
+
+// 7
+const currentMonth = 3;
+if ( currentMonth === 12 || currentMonth === 1 || currentMonth === 2 ) {
```

Comment:
```suggestion
if (currentMonth === 12 || currentMonth === 1 || currentMonth === 2) {
```
Please be consistent in code formatting